### PR TITLE
Create help docs for NT Key caching

### PR DESCRIPTION
**Pull request by stgmsa**
Created Docs for NT Key caching

# Description
Docs for NT key caching

# Impacts
Docs for NT key caching

# Delete branch after merge
YES

# Checklist
- [yes] Document the feature
- [no] Add OpenAPI specification
- [no] Add unit tests
- [no] Add acceptance tests (TestLink)

## Bug Fixes

# UPGRADE file entries
docs/installation/performance_optimizations.asciidoc

### CODE REVIEW

File: docs/installation/performance_optimizations.asciidoc [hunk 1]
```diff
@@ -18,6 +18,101 @@ endif::[]
...
+Otherwise, it will fallback to a traditional transitive login with the domain controller and cache the session key (NT Key) if successful authenticated.
+
+NOTE: The NT Key cache doesn't cache any NT hashes or user passwords, it caches the NT session key for each connected device for a period of time if enabled.
+
+NOTE: NT Key cache will also caches the wrong password count in order to block the device that keep trying using wrong password (or old password) to avoid an account lockout.
+
+
+==== PacketFence Configuration
+
+To Enable NT key caching, you'll have need a valid domain config entry in _Configuration -> Policies and Access Control -> Domains -> Active Directory Domains_
```
Comment (fdurand) [inline docs/installation/performance_optimizations.asciidoc]: you will need

File: docs/installation/performance_optimizations.asciidoc [hunk 2]
```diff
@@ -18,6 +18,101 @@ endif::[]
...
+
+At the bottom of the domain configuration page
+
+* Enable NT Key cache.
+* Specify a cache expiration time. Default is 3600 (in seconds)
```
Comment (fdurand) [inline docs/installation/performance_optimizations.asciidoc]: or a little bit more than the reauth period on the network equipment (Like 10800s we set on the cisco switch interface)

File: docs/installation/performance_optimizations.asciidoc [hunk 3]
```diff
@@ -18,6 +18,101 @@ endif::[]
...
+
+
+*Password reset* and *Password change* Events must be reported to ensure Windows NT Key cache works properly.
+Other events listed above will be optional but recommended.
+
+To attach the Windows Event Notifier, you'll need
+
+ * copy the windows event notifier powershell script in /usr/local/pf/addons/AD/password_change_notifier.ps1 to *each* domain controller you want to have NT Key cache enabled.
+ * Modify the script, Fill in the PacketFence server IP and username and password of Event Reporter.
+ * Go to windows event viewer, try to find an event of specific category. e.g. 4724 for password reset and 4723 for password change.
```
Comment (fdurand) [inline docs/installation/performance_optimizations.asciidoc]: IMO we should explain how to configure it via the "Task Scheduler" instead

File: docs/installation/performance_optimizations.asciidoc (hunk 2 above)
Comment (stgmsa) [reply docs/installation/performance_optimizations.asciidoc]: default set to 12000, changed in docs, will also change in admin UI and ntlm_auth_api python script

File: docs/installation/performance_optimizations.asciidoc (hunk 3 above)
Comment (stgmsa) [reply docs/installation/performance_optimizations.asciidoc]: Done. added detail steps and images as well.

**Pull request merged**